sdk-exporter/prometheus/src/main/scala/org/typelevel/otel4s/sdk/exporter/prometheus/PrometheusMetricExporter.scala

```diff
@@ -0,0 +1,251 @@
+    */
+  def serverBuilder[F[_]: Async: Network: Console]: HttpServerBuilder[F] =
+    new HttpServerBuilderImpl[F](
+      host = Defaults.Host,
+      port = Defaults.Port,
+      defaultAggregationSelector = AggregationSelector.default,
+      withoutUnits = Defaults.WithoutUnits,
+      withoutTypeSuffixes = Defaults.WithoutTypeSuffixes,
+      disableScopeInfo = Defaults.DisableScopeInfo,
+      disableTargetInfo = Defaults.DisableTargetInfo
+    )
+
+  private final case class BuilderImpl[F[_]: Concurrent: Console](
+      defaultAggregationSelector: AggregationSelector
+  ) extends Builder[F] {
+    def withDefaultAggregationSelector(
+        selector: AggregationSelector
+    ): Builder[F] = copy(defaultAggregationSelector = selector)
+
+    def build: F[MetricExporter.Pull[F]] =
+      Ref.empty[F, Vector[MetricProducer[F]]].map { ref =>
+        new PrometheusMetricExporter[F](
+          ref,
+          defaultAggregationSelector
+        )
+      }
+
+  }
+
+  private final case class HttpServerBuilderImpl[F[_]: Async: Network: Console](
+      host: Host,
+      port: Port,
+      defaultAggregationSelector: AggregationSelector,
+      withoutUnits: Boolean,
+      withoutTypeSuffixes: Boolean,
+      disableScopeInfo: Boolean,
+      disableTargetInfo: Boolean
+  ) extends HttpServerBuilder[F] {
+    def withHost(host: Host): HttpServerBuilder[F] =
+      copy(host = host)
+
+    def withPort(port: Port): HttpServerBuilder[F] =
+      copy(port = port)
+
+    def withDefaultAggregationSelector(
+        selector: AggregationSelector
+    ): HttpServerBuilder[F] = copy(defaultAggregationSelector = selector)
+
+    def withWithoutUnits(withoutUnits: Boolean): HttpServerBuilder[F] =
+      copy(withoutUnits = withoutUnits)
+
+    def withWithoutTypeSuffixes(withoutTypeSuffixes: Boolean): HttpServerBuilder[F] =
+      copy(withoutTypeSuffixes = withoutTypeSuffixes)
+
+    def withDisableScopeInfo(disableScopeInfo: Boolean): HttpServerBuilder[F] =
+      copy(disableScopeInfo = disableScopeInfo)
+
+    def withDisableTargetInfo(disableTargetInfo: Boolean): HttpServerBuilder[F] =
+      copy(disableTargetInfo = disableTargetInfo)
+
+    def build: Resource[F, MetricExporter.Pull[F]] =
+      Resource.eval(Ref.empty[F, Vector[MetricProducer[F]]]).flatMap { ref =>
+        val exporter = new PrometheusMetricExporter[F](
+          ref,
+          defaultAggregationSelector
+        )
+
+        val routes = PrometheusHttpRoutes
+          .routes[F](exporter, withoutUnits, withoutTypeSuffixes, disableScopeInfo, disableTargetInfo)
+
+        val metricOptions =
+          "{" +
+            s"withoutUnits=$withoutUnits, " +
+            s"withoutTypeSuffixes=$withoutTypeSuffixes, " +
+            s"disableScopeInfo=$disableScopeInfo, " +
+            s"disableTargetInfo=$disableTargetInfo}"
+
+        EmberServerBuilder
+          .default[F]
+          .withHost(host)
+          .withPort(port)
+          .withHttpApp(routes.orNotFound)
```

Comment:
By default, metrics should be exposed at `/metrics`:
```suggestion
          .withHttpApp(Router("metrics" -> routes).orNotFound.orNotFound)
```

https://github.com/prometheus/client_java/blob/main/prometheus-metrics-exporter-httpserver/src/main/java/io/prometheus/metrics/exporter/httpserver/HTTPServer.java#L71-L75

Reply:
Done.